client/components/purchase-detail/purchase-button.jsx

```diff
@@ -23,7 +24,7 @@ const PurchaseButton = ( {
 			rel={ rel }
 			primary={ primary }
 		>
-			{ text }
+			{ text } { icon && <Gridicon icon={ icon } /> }
```

Comment:
Instead of passing an icon down to this component, I think we should be able to use the `isOutsideCalypso()` helper method to render that icon:

```suggestion
			{ text } { isOutsideCalypso( href ) && <Gridicon icon="external" /> }
```

Reply:
Updated, thanks a lot for the suggestion.